Tests/Functional/AbstractFunctionalTestCase.php

```diff
@@ -51,65 +48,278 @@
  */
 abstract class AbstractFunctionalTestCase extends FunctionalTestCase
 {
-    protected array $testExtensionsToLoad = ['typo3conf/ext/pdfviewhelpers'];
+    protected array $testExtensionsToLoad = [
+        'bithost-gmbh/pdfviewhelpers',
+    ];
 
     /**
      * 150 words 890 characters
      */
     protected string $loremIpsumText = 'Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.';
 
-    protected array $typoScriptFiles = [];
+    private array $baseTypoScript = [
```

Comment:
I actually also thought about defining the TypoScript like this to fix the tests, the advantage being that the tests would become independent of the TYPO3 part responsible for loading TypoScript. However I then found a more or less simple way to keep the TypoScript files and load them. Although not perfect, I think I would stick with it for the time being: https://github.com/bithost-gmbh/pdfviewhelpers/commit/ed62bba99fc069bddc3addc5fad3fd71534ff2db

Reply:
I tried the TYPO3 Core way. They also inject TypoScript in the class directly. Without relating to a file.

Reply:
Thanks for the info! Here I am really open for doing it either way 👍